.rubocop.yml

```diff
@@ -100,6 +100,9 @@ Rails/TimeZone:
     - strict
     - flexible
 
+Rails/Delegate:
+  Enabled: false
```

Comment:
What is it claiming could be delegated? 🤔

Reply:
it wanted

```ruby
  delegate :two_factor_enabled?, to: :mfa_user

  # instead of
  def two_factor_enabled?
    mfa_user.two_factor_enabled?
  end
```

Reply:
Ah!